common/laws/00_citizenship.txt

```diff
@@ -305,6 +309,7 @@ law_true_multicultural = {
 		country_infamy_generation_mult = -0.10
 		country_bureaucracy_investment_cost_factor_mult = 0.25
 		state_assimilation_mult = 0.25
+		country_leverage_resistance_mult = -0.25
```

Comment:
At this rate wouldn't a really liberal country have like -1 leverage resistance multiplier?